source/configuration_manual/shared_mailboxes/cluster_setup.rst

```diff
@@ -58,15 +65,20 @@ in this mailbox ``someone1`` would access a mailbox with the name
 .. note:: See :ref:`user_shared_mailboxes_vs` for an explanation more details on the used variables.
 
 Additionally imapc must be configured accordingly on the backends:
- * :ref:`setting-imapc_master_user` must be set to ``%u`` e.g. the logged in
-   user becomes imapc master user
+ * :ref:`setting-imapc_master_user` must be set to ``%{userdb:sharing_user}``
+   this requires to return to the ``sharing_user`` field for backend userdb
+   lookups and thereby use it as imapc_master_user=%{userdb:sharing_user}.
```

Comment:
A bit confusing/repetitive sentence. Also where does sharing_user come from? Maybe %{master_user}? Something like this? :

```
  * Master passdb must return ``userdb_sharing_user=%{master_user}``
  * :ref:`setting-imapc_master_user` must be set to ``%{userdb:sharing_user}``.
    This makes sure ...
```

(not sure if that's the right way)